include/gazebo_mavlink_interface.h

```diff
@@ -236,6 +235,7 @@ class GazeboMavlinkInterface : public ModelPlugin {
   int _fd;
   struct sockaddr_in _myaddr;     ///< The locally bound address
   struct sockaddr_in _srcaddr;    ///< SITL instance
+  struct sockaddr_in _hil_gps_addr;
```

Comment:
what's the purpose of this socket address?

Reply:
to emulate GPS receiver directly connected to on-board processor

Reply:
This needs a correspondent comment explaining this.